### PR TITLE
change view when clicking on a screenshot

### PR DESCRIPTION
**Pull request by ruben1**

### CODE REVIEW

**Comment by AndrewSouthpaw** [comment]
There's a couple lint errors, so those should be fixed.

File: client/app/src/js/controllers/screenshot.js [hunk 1]
```diff
@@ -5,13 +5,22 @@
  */
 
 function ScreenshotController(Screenshot, User) {
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshot.js]: It'd be nice to have documentation here. Explain the why of this controller.

File: client/app/src/js/controllers/screenshots.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * Handles display of a user screenshots.
  */
 
-function ScreenshotsController(screenshots, Screenshot) {
+function ScreenshotsController(screenshots, Screenshot, $location) {
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshots.js]: Place angular dependencies first, per Jackson's recommendation

File: client/app/src/js/controllers/screenshots.js [hunk 3]
```diff
@@ -13,18 +13,22 @@ function ScreenshotsController(screenshots, Screenshot) {
         console.log(data);
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshots.js:13]: Push results to `this.screenshots`? If deliberately incomplete, leave a TODO

File: client/app/src/js/controllers/screenshots.js [hunk 4]
```diff
@@ -13,18 +13,22 @@ function ScreenshotsController(screenshots, Screenshot) {
...
+  this.changeView = function(screenshotId) {
+    $location.path('/screenshot/' + screenshotId);
+  }
 }
 
 ScreenshotsController.$inject = [
   'screenshots',
-  'Screenshot'
+  'Screenshot',
+  '$location'
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshots.js]: move to be first

File: client/app/src/js/controllers/screenshots.js [hunk 5]
```diff
@@ -13,18 +13,22 @@ function ScreenshotsController(screenshots, Screenshot) {
...
+  this.getScreenshots = function(id){
+    Screenshot.getScreenshots(id)
       .success(function(data){
         console.log(data);
+        this.screenshots = data;
       });
-
   };
+  this.changeView = function(screenshotId) {
+    $location.path('/screenshot/' + screenshotId);
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshots.js:24]: I think we want to make the path `/user/:username/screenshot/:screenshotId`?

**Comment by AndrewSouthpaw** [comment]
added some comments for you.

File: client/app/src/js/controllers/screenshots.js (hunk 3 above)
Comment (ruben1) [inline client/app/src/js/controllers/screenshots.js:13]: "this.screenshots" is set to "screenshots" that we inject directly to the controller.

**Comment by AndrewSouthpaw** [comment]
LGTM.

**Pull request merged**